src/libraries/System.Text.Encoding.CodePages/src/System.Text.Encoding.CodePages.csproj

```diff
@@ -91,6 +91,7 @@ System.Text.CodePagesEncodingProvider</PackageDescription>
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">
     <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
     <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="$(SystemRuntimeCompilerServicesUnsafeVersion)" />
+    <PackageReference Include="System.ValueTuple" Version="$(SystemValueTupleVersion)" />
```

Comment:
Interesting. We just published a new version for this in maintenance-packages. We still need to upload it to nuget and consume it in our runtime infra, but it's there.

Reply:
Need to be done globally to update SystemValueTupleVersion I guess.

Reply:
Yep, I take care of that. Don't worry.